plotly/plotly/plotly.py

```diff
@@ -869,7 +868,7 @@ def _fill_in_response_column_ids(cls, request_columns,
         for req_col in request_columns:
             for resp_col in response_columns:
                 if resp_col['name'] == req_col.name:
-                    req_col.id = '{0}/{1}'.format(grid_id, resp_col['uid'])
+                    req_col.id = '{0}:{1}'.format(grid_id, resp_col['uid'])
```

Comment:
Got it. That's weird, because this definitely worked at some point. Here is the section and example in our docs: 
![image](https://cloud.githubusercontent.com/assets/1280389/20536085/aa065254-b0b5-11e6-916d-03678f360502.png)